backend/config/curation-api.yml

```diff
@@ -356,7 +356,37 @@ paths:
           $ref: "#/components/responses/401"
         "404":
           $ref: "#/components/responses/404"
-
+  /v1/checks/submission_key:
```

Comment:
I think I like `/v1/verify/` more than `/v1/checks/`

What do you think?

Reply:
I considered using verify. However, I am avoiding the use of verbs in the URL path as is the REST way.